package.json

```diff
@@ -37,14 +37,14 @@
     "@seadub/danger-plugin-junit": "^0.2.0",
     "@seadub/danger-plugin-titanium-module": "1.0.0",
     "clang-format": "^1.2.3",
-    "commitizen": "^4.0.3",
-    "danger": "^10.4.0",
-    "eslint": "^7.9.0",
-    "eslint-config-axway": "^6.0.0",
+    "commitizen": "^4.2.4",
+    "danger": "^10.6.6",
+    "eslint": "^7.32.0",
+    "eslint-config-axway": "^6.0.2",
     "eslint-plugin-jasmine": "^4.1.0",
     "husky": "^7.0.2",
-    "jasmine-core": "^3.3.0",
-    "karma": "^5.2.2",
+    "jasmine-core": "^3.9.0",
+    "karma": "^6.3.4",
```

Comment:
We have to stick on 5.x of karma as 6.x is incompatible. If I remember correctly it's down to them upgrading to v3 socket.io